userspace/libsinsp/fdinfo.cpp

```diff
@@ -39,6 +39,10 @@ template<> void sinsp_fdinfo_t::reset()
 {
 	m_type = SCAP_FD_UNINITIALIZED;
 	m_flags = FLAGS_NONE;
+	if(m_callbaks)
```

Comment:
If I'm not wrong we can avoid this check because delete should be safe on nullptr.